advanced-usage/index.md

```diff
@@ -126,137 +148,110 @@ conveniently accessing available pages, tools, and services.
 
-### Log Browser
-{{ service(service="UAV LogViewer", link="https://ardupilot.org/copter/docs/common-uavlogviewer.html") }}
+{% pirate() %}
+### Available Services
 
-- Built in log viewer for powerful analysis of vehicle telemetry
-- Currently only set up to fetch logs automatically from Linux-based autopilots
-   - e.g. Pixhawk **not** yet supported
+The Available Services page provides developer access to the underlying http
+server interfaces of the services upon which BlueOS is based. Each service is
+listed with 
+- the port it is served at
+- a meaningful name
+- a web-page link using the active 
+[network configuration](../getting-started/#interface-access)
 
-{{ easy_image(src="log-browser", width=600) }}
+and where relevant
+- its API documentation (in a live-testable form)
+- the current API version
 
-### Video
-{{ service(service="MAVLink Camera Manager", link="https://github.com/bluerobotics/mavlink-camera-manager/", port=6020) }}
+Any service that provides an http server with a
+[title tag](https://www.w3schools.com/tags/tag_title.asp) will be displayed.
+Documentation can also be parsed if
+- it follows the swagger/open api spec, and 
+- is available at `/docs` or `/v1.0/ui`
+{% end %}
+{{ easy_image(src="available-services", width=600, class="pirate") }}
 
-- BlueOS automatically detects H264-encoded video streams
-- The first time BlueOS starts up it will auto-configure any cameras that are 
-connected at that time, with UDP streams counting up from port `5600`
-   - e.g. a second camera at first startup would be streamed to port `5601`
-   - Auto-configuration also occurs if the [settings are reset](#settings)
 {% pirate() %}
-      - It's also possible to manually reset _only_ the camera settings by deleting
-      the file `/root/.config/mavlink-camera-manager/settings.json` via
-      [the file browser](#file-browser) or [the terminal](#terminal), or starting
-      the camera manager inside the tmux session with the `--reset` flag
+### File Browser
 {% end %}
+{{ service(service="File Browser", port=7777, link="https://github.com/filebrowser/filebrowser") }}
+{% pirate() %}
 
-- After the initial startup, settings are saved and persistent across reboots
-   - Further changes require manually re-configuring streams
-   - New streams need to be manually added
-      - The stream endpoint should be set to `udp://<surface-IP>:<port>`
-      - e.g. `udp://192.168.2.1:5601`
-- The streams are also presented via MAVLink, so QGroundControl (v4.1.7) can
-toggle between them without needing to know specific ports.
-<div style="text-align:center;">
+The File Browser allows viewing, editing, downloading, and uploading BlueOS files.
 
-{{ easy_image(src="qgc_switch_streams", width=400) }}
+{% end %}
+{{ easy_image(src="file-browser", width=600, class="pirate") }}
 
-</div>
+### Log Browser
+{{ service(service="UAV LogViewer", link="https://ardupilot.org/copter/docs/common-uavlogviewer.html") }}
```

Comment:
I'm not sure this is quite the right link...

Reply:
Why? Is there another link with documentation for Log Viewer?

Reply:
Just that it document logviewer instead of the logbrowser

Reply:
Yeah, for the pages that don't have a dedicated backend service I just try to come up with whatever relevant link I can. In this case I think I figured the log viewer was the biggest feature of the browser, so it was an ok thing to link to.

Feel free to let me know if there's a better place/thing to link to :-)

Reply:
Fair enough